packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -59,8 +59,11 @@ export default function (server: Server, ctx: AppContext) {
               .whereRef('post_embed_image.postUri', '=', 'feed_item.postUri'),
           )
       } else if (filter === 'posts_no_replies') {
-        // only posts, no replies
-        feedItemsQb = feedItemsQb.where('post.replyParent', 'is', null)
+        feedItemsQb = feedItemsQb.where((qb) => {
```

Comment:
such a nit, but for these internal qb, we usually leave out the brackets & the return